deps/wazuh_testing/wazuh_testing/remote.py

```diff
@@ -576,12 +586,13 @@ def intercept_socket_data(data):
         control_service('start', daemon='wazuh-analysisd')
 
 
-def check_agent_received_message(message_queue, search_pattern, timeout=5, update_position=True, error_message=''):
+def check_agent_received_message(message_queue, search_pattern, escape=False, timeout=5, update_position=True, error_message=''):
```

Comment:
New parameters must be added at the end to avoid collateral damages.

Reply:
Done in a3680eda20c727382a4c272a7e914c96689417d9